.github/workflows/create_release.yml

```diff
@@ -0,0 +1,55 @@
+name: Create GitHub Release for Extension Package
+
+on:
+    workflow_dispatch:
+
+jobs:
+    release:
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkout code
+              uses: actions/checkout@v3
+
+            - name: Install Node.js
+              uses: actions/setup-node@v3
+              with:
+                  node-version: '16.x'
+
+            - name: Install dependencies
+              run: npm install
+
+            - name: Build VSIX Package
+              run: npx vsce package
+
+            - name: Create release
+              id: create_release
+              uses: actions/github-script@v6
+              with:
+                  script: |
+                      const fs = require('fs');
+                      const packageVersion = require('./package.json').version;
+                      const response = await github.rest.repos.createRelease({
+                        owner: context.repo.owner,
+                        repo: context.repo.repo,
+                        tag_name: `v${packageVersion}`,
+                        name: `Release ${packageVersion}`,
+                        body: `Generating release for version ${packageVersion}`
+                      });
+                      return response.data.id;
+            - name: Attach VSIX package to release
+              uses: actions/github-script@v6
+              with:
+                  script: |
+                      const fs = require('fs');
+                      const packageJson = require('./package.json');
+                      const packageName = packageJson.name;
+                      const packageVersion = packageJson.version;
+                      const vsixFilename = `${packageName}-${packageVersion}.vsix`;
+                      const uploadUrl = await github.rest.repos.uploadReleaseAsset({
```

Comment:
nit: Do we need `uploadUrl` if we're not returning its property, something like we co in `Create Release` with `return response.data.id`?

Reply:
Ah, good catch. No. I had started this script anticipating more steps to come after, then didn't do that. I'll take it out.